tests/test_rut_crypto_utils.py

```diff
@@ -51,3 +52,48 @@ def test_get_subject_rut_from_certificate_pfx_fails_if_rut_info_is_missing(self)
                     password=password,
                 )
             self.assertEqual(cm.exception.args, ('Certificate has no RUT information',))
+
+    def test_get_subject_rut_from_certificate_pfx_does_not_throws_attribute_error_if_has_object_without_type_id(  # noqa: E501
```

Comment:
_"throws"_ should be in singular form because it comes after _"does"_.

```suggestion
    def test_get_subject_rut_from_certificate_pfx_does_not_throw_attribute_error_if_has_object_without_type_id(  # noqa: E501
```